jujugui/static/gui/src/test/test_ui_state.js

```diff
@@ -1033,7 +1033,7 @@ describe('UI State object', function() {
         sectionB: {}
       }
     }, {
-      '/inspector/service123/unit/13/': {
+      '/inspector/service123/13/': {
```

Comment:
To bring this test up to the new functionality the metadata should contain `activeComponent: 'unit'` and then I think the original url is correct.